### ISSUE TITLE
HTMLの文法違反

### ISSUE THREAD

**Issue by magi1125** [tokyo-metropolitan-gov/covid19#6750]
2021年9月4日〜9月13日までに実施された、アクセシビリティ試験結果に基づくIssueです。
達成基準4.1.1に対して不適合

## 改善詳細 / Details of Improvement

対象：全ページ

https://validator.w3.org/nu/ のチェック結果のうち、warningになっていて、かつ入れ子ルールやroleやariaなどの支援技術の動作に影響がありそうなものをピックアップ。

A document must not include more than one meta element with a charset attribute.
`<meta data-n-head="ssr" charset="utf-8">`
`<meta data-n-head="ssr" data-hid="charset" charset="utf-8">`
meta charsetが複数ある

Element div not allowed as child of element a in this context. (Suppressing further errors from this subtree.)
`<div class="SideNavigation-HeaderText" data-v-32e6c11e="">`
h1 > a > divになっていて入れ子ルール違反

Element div not allowed as child of element button in this context. (Suppressing further errors from this subtree.)
`<div class="v-expansion-panel-header__icon">`
button > divになっていて入れ子ルール違反

Element style not allowed as child of element div in this context. (Suppressing further errors from this subtree.)
noscript内部のstyle要素が入れ子ルール違反

Element div is missing one or more of the following attributes: role.
`<div aria-expanded="false" class="v-expansion-panel">`
aria-expandedはroleがない要素には指定できない（aria-expandedはbutton側だけにつける）

Element div not allowed as child of element button in this context. (Suppressing further errors from this subtree.)
`<button><div style="background-color: rgb(XXX); border-color: rgb(XXX);">`
グラフの凡例。button > divになっていて入れ子ルール違反

The role attribute must not be used on a th element which has a table ancestor with no role attribute, or with a role attribute whose value is table, grid, or treegrid.
`<th role="columnheader" scope="col" aria-label="地域" class="text-start">`
role="columnheader" があるとルール違反（親要素が普通のtableのときには使えない）

Attribute justify-center not allowed on element div at this point.Attribute align-center not allowed on element div at this point.
`<div class="v-overlay v-overlay--absolute theme--dark" justify-center="" align-center="" style="z-index: 5;">`
陽性者の属性のテーブル送り周り。謎の属性がある

The element input must not appear as a descendant of an element with the attribute role=button.
`<input aria-label="1ページ当たり" id="input-918" readonly="readonly" type="text" autocomplete="off">`
陽性者の属性のテーブル送り周り。role=button > input になっていて入れ子ルール違反

## 期待する見せ方・挙動 / Expected behavior
文法違反を解消する

## 動作環境・ブラウザ / Environment
すべて

**Comment by kaizumaki** [comment]
こちら対応します。

**Comment by kaizumaki** [comment]
> Element div is missing one or more of the following attributes: role.
> `<div aria-expanded="false" class="v-expansion-panel">`
> aria-expandedはroleがない要素には指定できない（aria-expandedはbutton側だけにつける）

以上を残して、他は #6770 で修正しました。
以下コメントを転記します。

> `div aria-expanded="false"`は、#6756 #6757 #6760 の共通課題ですよね。文法違反はそれはそれでありつつ、どっちかというとdivについていてbuttonについていない状況だとスクリーンリーダーがこれを折り畳みだとわからない点があり、どうしたものかなーという感じです。

_Originally posted by @magi1125 in https://github.com/tokyo-metropolitan-gov/covid19/issues/6770#issuecomment-922909538_
